scripts/Install-VSCode.ps1

```diff
@@ -100,22 +130,50 @@ param(
 )
 
 if (!($IsLinux -or $IsOSX)) {
-
-    $codeCmdPath = "C:\Program Files (x86)\Microsoft VS Code\bin\code.cmd"
-
+    switch ($Architecture) {
+        "64-bit" {
+            if ((Get-CimInstance -ClassName Win32_OperatingSystem).OSArchitecture -eq "64-bit") {
+                $codePath = $env:ProgramFiles
+                $bitVersion = "win32-x64"
+            }
+            else {
+                $codePath = ${env:ProgramFiles(x86)}
+                $bitVersion = "win32"
+                $Architecture = "32-bit"
+            }
+            break;
+        }
+        "32-bit" {
+            $codePath = ${env:ProgramFiles(x86)}
```

Comment:
Oops, missed this ... sorry.  I'm pretty sure on a 32-bit OS you'd use just ${env:ProgramFiles}.  ${env:ProgramFiles(x86)} is unique to the 64-bit version of Windows.  See https://en.wikipedia.org/wiki/Environment_variable#Windows

Reply:
oh yes. of course. I'll add in the idiot proofing for 32-bit on 64-bit now.